View/ViewHandler.php

```diff
@@ -448,6 +449,18 @@ private function initResponse(View $view, $format)
             if ($serializer instanceof SerializerInterface) {
                 $context = $this->getSerializationContext($view);
                 $content = $serializer->serialize($data, $format, $context);
+            //       test for symfony 2.7 version 
+            } elseif (class_exists('\Symfony\Component\Serializer\Normalizer\ObjectNormalizer') && $serializer instanceof SymfonySerializerInterface) {
+                    
+                $context = $this->getSerializationContext($view);
```

Comment:
@lsmith77 the issue with the View API is that it currently exposes some methods dealing with the JMSSerializer class directly. This means that supporting any other serializer needs to provide another way to configure them.

IMO, the clean solution would be to provide a way to configure the serializer without dealing with the JMSSerializer classes directly (except maybe inside a callable passed by the user, which would receive the serializer and decide which logic to apply). this way, we could build an API which does not depend on JMSSerializer.